doc/types/verify_info.qbk

```diff
@@ -0,0 +1,226 @@
+[/
+    Copyright (c) 2013-2016 Vinnie Falco (vinnie dot falco at gmail dot com)
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+]
+[section:nudb__verify_info verify_info]
+Describes database statistics calculated by [link nudb.ref.nudb__verify `verify`].
```

Comment:
We should not manually write out that which Doxygen can automatically extract and generate. The existing workflow is fine:
http://vinniefalco.github.io/nudb/nudb/ref/verify_info.html

I like the improvements made to the descriptions but they need to happen in the Javadoc comment so that there are not two copies of the doc and so that its easy to update the comment when the signature changes (since they are adjacent in the same file).